### ISSUE TITLE
All Criterias Deleted....

### ISSUE THREAD

**Issue by Omuck3** [joshiejack/Progression#61]
My laptop died while I was editing, and all my criterias are gone.  I know I can't get them back, but is there any way for me to prevent this in the future?  The tabs remained.

**Comment by K-4U** [comment]
Save every so often?

**Comment by Omuck3** [comment]
I had added barely anything new.  Most of the stuff was already there.  But now it's all gone.

**Comment by joshiejack** [comment]
What version was this. Because in the 1.8/1.9 versions there are backup files made everytime it saves.

**Comment by Omuck3** [comment]
It was the 1.7 version.  The weird part is all the old stuff deleting...  Because I could understand the newer stuff being deleted, but not the old stuff.

**Comment by joshiejack** [comment]
Oh, then you're out of luck. You'll need to make your own backups.

**Comment by Omuck3** [comment]
Ok.

**Comment by BLSK** [comment]
So Omuck knows:

They will delete anyways... there is a bug I found that while I was playing it just deleted the Criteria... no crash no nothing... it was just gone and never found out what caused it(couldn't find anywhere).  1.7.10's version is not advisable and I mentioned it to joshiejack a few months ago.  I am trying to destroy 1.9's as I plan to finish my modpack in it and if I find the same issue(hopefully not :'( so many hours lost XD) I will be hopefully able to bring it to attention(along with other people hopefully too :) but it was a new mod so anything expected, its almost been 2 years since I started this pack... fearful it may never work, needing 3 more mods before version 2.0.0 is ready).

just a suggestion... sadly give up on 1.7 if you intend to make quests... you will regret using it over and over.

and don't worry josh...  I am trying to see if I can break it :(... hopefully not :)
